bindings/pyroot/pythonizations/test/CMakeLists.txt

```diff
@@ -127,6 +127,13 @@ endif()
 
 # RTensor pythonizations
 if (dataframe AND tmva)
+    if(NOT MSVC OR CMAKE_SIZEOF_VOID_P EQUAL 4 OR win_broken_tests)
+        ROOT_ADD_PYUNITTEST(pyroot_pyz_sofie_gnn sofie_gnn.py PYTHON_DEPS numpy)
```

Comment:
The test doesn't only depend on numpy, I can clearly see both `graph_nets` and `sonnet` packages being imported. Be very careful about this!

Reply:
made the change, thanks!